components/app/landing/LandingDaoProposals.vue

```diff
@@ -5,7 +5,7 @@
     <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2 xl:grid-cols-4 2xl:grid-cols-4 gap-4 mt-10">
       <div v-for="project in projects" :key="project.title">
         <NuxtLink to="/project-overview">
```

Comment:
issue, question(blocking): Nuxt.js automatically includes smart prefetching. That means it detects when a link is visible, either in the viewport or when scrolling and prefetches the JavaScript for those pages so that they are ready when the user clicks the link. Are you sure that you want to pre-fetch every page here? 
I am not so sure if this also includes any graphics but if yes then we should disable it. Otherwise this feature is fine as this will be only done when the browser is idle and our server can handle sending a bundled js file to the user. If this also includes other assets we should disable it.